app/services/virtual-webcam.ts

```diff
@@ -115,9 +134,23 @@ export class VirtualWebcamService extends StatefulService<IVirtualWebcamServiceS
   }
 
   update(type: VCamOutputType, name: string) {
+    this.virtualCamSettings.db.write(() => {
+      this.virtualCamSettings.deepPatch({
+        outputType: type,
+        outputSelection: name,
+      });
+    });
     obs.NodeObs.OBS_service_updateVirtualCam(type, name);
   }
 
+  outputType(): VCamOutputType {
```

Comment:
Not sure if you would like this be reactive? If so, we use `get outputType()`, but the function call also works if you don't want reactivity